### ISSUE TITLE
valueSuffix misaligned from progressValueStyle, multiple on same page causes gradient issues (react-native-web)

### ISSUE THREAD

**Issue by PrestonFocusV** [nithinpp69/react-native-circular-progress-indicator#90]
Unlike on native platforms, when adding a suffix on web it appears far away from the progress value due to the input width.
Also, having multiple on the same screen with different gradients doesn't appear to be working. Both inherit the same gradient.

**Comment by PrestonFocusV** [comment]
@nithinpp69 Are you able to reproduce these issues on web?

**Comment by nithinpp69** [comment]
Yes. This might have been introduced with [v4.2.0](https://github.com/nithinpp69/react-native-circular-progress-indicator/releases/tag/V4.2.0). With this, the prefix and suffix components were separated from the progress value and are rendered with separate Text components. The progress value on the other hand uses a TextInput to display. On web, it seems like the progress value Text input is having a lot of padding by default and the `padding:0` is not adding any value there.

**Comment by ArchanaVasumathi** [comment]
Are you able to resolve this issue. I am also facing similar issue with expo sdk 47

**Comment by PrestonFocusV** [comment]
@nithinpp69 Is there any plan to fix this? Is the project still being maintained?
